packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -169,8 +174,8 @@ export const ItemUI = styled.li`
 `;
 
 export const ItemBadgeUI = styled.span`
-	margin-left: ${ ( props ) => ( props.isRTL ? '0' : '8px' ) };
-	margin-right: ${ ( props ) => ( props.isRTL ? '8px' : '0' ) };
+	margin-left: ${ ( props ) => ( props.isRTL ? '0' : space( 1 ) ) };
+	margin-right: ${ ( props ) => ( props.isRTL ? space( 1 ) : '0' ) };
 	display: inline-flex;
 	padding: 4px 12px;
```

Comment:
Can we use `space` here too?

Reply:
Ah my mistake David. I didn't see these comments because I was focused on getting the e2e tests to pass, but I agree that they should be switched to `space`.

Reply:
I'm going to open up a quick PR to update all pixel values to use `space`

Reply:
https://github.com/WordPress/gutenberg/pull/26523